openslides_backend/action/mixins/motion_meeting_user_create.py

```diff
@@ -18,7 +18,7 @@
 
 
 def build_motion_meeting_user_create_action(
```

Comment:
Why do you want to create a class at run time?

Reply:
This function is used to create base classes for about four actions for motion-related meeting_user-models.
Therefore I don't think it is called that often.

I am merely editing this, because I need some of these actions changed.

If you need further information on how this all works, why don't you look at [this commit](https://github.com/OpenSlides/openslides-backend/commit/218ca391bcd8c628429c1c58e90ced987a3733ed), _where @jsangmeister wrote this code_